scripts/build-dist.sh

```diff
@@ -16,25 +17,33 @@ case "$(uname -s)" in
     ;;
 esac
 
+version=`node -e "var fs=require('fs');console.log(JSON.parse(fs.readFileSync('package.json')).version)"`
```

Comment:
This can be simplified:
```
version=`node -p -e "require('./package.json').version"`
```

This is what we do in `release-branch.sh` for example: https://github.com/yarnpkg/yarn/blob/9347540b09cc71d71f8651b2423ff8bbe3eeaf50/scripts/release-branch.sh#L6-L6